mlrun/db/httpdb.py

```diff
@@ -333,6 +333,10 @@ def connect(self, secrets=None):
                 server_cfg.get("default_tensorboard_logs_path")
                 or config.default_tensorboard_logs_path
             )
+            config.default_function_pod_resources = (
+                server_cfg.get("default_function_pod_resources")
+                or config.default_function_pod_resources
+            )
```

Comment:
Note the comment in line 302, it's not true for this one, it does not have default value, therefore makes more sense to prioritize client first
Also you're doing `or` on the Config here (the client part) it will always be true even if it doesn't have values set